tests/test_regularizers.py

```diff
@@ -161,6 +162,23 @@ class NormedL2RegularizerTest(_LpRegularizerTest, unittest.TestCase):
 
     regularizer_kwargs = {'normalize': True, 'p': 2}
 
+    @pytest.mark.slow
```

Comment:
these tests are slow?

Reply:
Likely not :sweat_smile: 

However, they are statistically testing, i.e. they are not guaranteed to pass, but only in the majority of cases.

Reply:
Maybe we can also do something with [1-sample t-test](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.ttest_1samp.html) ?

Reply:
And add another test annotation for statistical tests, whose failure is not critical?